### DISCUSSION TITLE
Splitting to multi page pdf dcuments in R

### DISCUSSION THREAD

**Discussion by saracv** [qpdf/qpdf#1057] [Q&A]
Hello, 
I am currently using this package https://cran.r-project.org/web/packages/qpdf/qpdf.pdf which has helped me split pdfs into single page pdfs documents in the past. I need to split the documents into new PDF documents every two pages. Is there an argument I can use to set that? Thaks in advance for any help.

**Comment by jberkenbilt** [comment]
With the qpdf cli, you can say `qpdf --split --pages=2` to spit in groups of two pages. I'm not familiar with R the cran R project, but maybe this can give you ideas.
